### DISCUSSION TITLE
امکان تغییر پورت ssh

### DISCUSSION THREAD

**Discussion by Abbas-Firouzi** [hiddify/Hiddify-Manager#566] [Q&A سوالات رایج]
سلام
امکان تغییر پورت پیش فرض ssh
 یا اسنفاده از چند پورت رو توضیح میدید من هرکاری می کنم پورت نامبر دیفالت تغییر نمیکنه

**Comment by foroudia** [comment]
سلام
این دستور رو بزن تا فایل کانفیگ باز شه :
nano /etc/ssh/sshd_config

حالا دنبال کلمه پورت بگرد که به صورت کامند هستش، میتونی با برداشتن تگ کامند و تغییر عدد، پورت رو عوض کنی.
اگر هم میخوای چندتا پورت باز کنی برای ssh
باید همون خط رو دوباره بنویسی به عدد دیگه ای
